DoublyLinkedList/Model/DoublyLinkedItem.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Runtime.InteropServices.JavaScript.JSType;
+
+namespace DoublyLinkedList.Model;
+/// <summary> Ячейка двусвязного списка.</summary>
+internal class DoublyLinkedItem<T>
+{
+    #region Поля
+    /// <summary>Данные хранимые в ячейке списка</summary>
+    private T _data;
+    /// <summary>Ссылка на предыдущий элемент списка</summary>
+    internal DoublyLinkedItem<T> _previous;
+    /// <summary>Ссылка на следующий элемент списка</summary>
+    internal DoublyLinkedItem<T> _next;
+    #endregion
+
+    #region Свойства
+    /// <summary>Данные хранимые в ячейке списка</summary>
+    public T Data
+    {
+        get { return _data; }
+        set 
+        {
+            if(value == null)          
+                throw new ArgumentNullException(nameof(value), "Значение не может быть null. ");           
+            _data = value;
+        }
+    }
+
+    /// <summary>Ссылка на предыдущий элемент списка</summary>
+    public DoublyLinkedItem<T> Previous
+    {
+        get { return _previous; }
+        //set { _previous = value; }
+    }
```

Comment:
Можно было заменить на авто-свойства.
```suggestion
    public DoublyLinkedItem<T> Previous { get; internal set; }
```

Reply:
Но должна быть возможность изменять значение этого свойства при изменении состояния списка (при добавлении, удалении элементов).